pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -763,6 +763,12 @@ func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, rsOp
 		return true, nil
 	}
 
+	if volumeToMount.VolumeSpec != nil &&
+		volumeToMount.VolumeSpec.InlineVolumeSpecForCSIMigration {
+		klog.V(4).Infof("This volume %s is a migrated inline volume and is not resizable", volumeToMount.VolumeName)
```

Comment:
I'm unsure how we would even get here since resizing is only triggered through the presence of a PV?

Reply:
The resize function is called directly after `MountDevice`:
https://github.com/kubernetes/kubernetes/blob/d4d8daea7368e5fc9ad8c375efff74a65d0f91bf/pkg/volume/util/operationexecutor/operation_generator.go#L680

I have no idea the reasons why, but it is.